.golangci.yml

```diff
@@ -31,6 +31,7 @@ linters-settings:
   errorlint:
     # these are still common in Go: for instance, exit errors.
     asserts: false
+    errorf: false
```

Comment:
could we have ignored this on a single line? `//nolint:...` ?
which change prompted this?

Reply:
The lint rule forces the use of `%w` in error messages. That's a bad thing because you don't always want the original error to be exposed from your package API. For example, if I return a Kubernetes specific error from my Kube backend and a frontend becomes dependent on that, then we switch to a different backend that doesn't throw that same error we have misaligned behavior.

Allowing the use of `%v` reduces the scope of Hyrum's law.